libs/atomic-elements/src/lib/components/PositiveNegativeArea/PositiveNegativeArea.tsx

```diff
@@ -1,32 +1,15 @@
-import React from 'react';
-import * as Highcharts from 'highcharts';
+import Highcharts from 'highcharts';
 import HighchartsReact from 'highcharts-react-official';
 import Loader from '../Loader/Loader';
+import React from 'react';
+import { generalBaseOptions } from '../generalBaseOptions';
+import merge from 'lodash.merge';
 
-const baseOptions: Highcharts.Options = {
-  title: {
-    text: '',
-  },
-  credits: {
-    enabled: false,
-  },
-  legend: {
-    enabled: false,
-  },
-  chart: {
-    type: 'area',
-    backgroundColor: 'var(--transparent)',
-  },
-  yAxis: {
-    title: {
-      text: null,
-    },
-  },
+const baseOptions: Highcharts.Options = merge({}, generalBaseOptions, {
```

Comment:
I was wondering if the empty `{}` is necessary
```suggestion
const baseOptions: Highcharts.Options = merge(generalBaseOptions, {
```

Reply:
it is, I teste without initially, but merge modifies the reference, and doesnt create a new object everytime